project/src/main/world/chattable-manager.gd

```diff
@@ -201,20 +201,20 @@ func focused_chattable_creature_id() -> String:
 
 
 """
-Remove the specified creature from the '_creatures_by_id' mapping.
+Add the specified creature to the '_creatures_by_id' mapping.
```

Comment:
Grammatically, these comments should say 'adds', 'removes' and not 'add', 'remove'.

Reply:
Changed to adds/removes.